frontend/src/helpers/sign/open-facebook.ts

```diff
@@ -0,0 +1,11 @@
+import { ENV } from '@common/enums/app/env.enum';
+
+const openFacebookAuthPage = (token: string | null = null): void => {
+  window.open(
+    `${ENV.API_PATH}/auth/facebook/sign?token=${token}`,
+    'Facebook OAuth',
+    'height=615,width=605',
+  );
+};
```

Comment:
This function almost duplicates the google auth opener. Extract the reusable function